astropy/coordinates/baseframe.py

```diff
@@ -931,7 +931,7 @@ def replicate(self, copy=False, **kwargs):
 
         Returns
         -------
-        frameobj : same as this frame
+        frameobj : `BaseCoordinateFrame` subclass instance
```

Comment:
To clarify: the current frame-like is an instance *or* a string, which is sometimes (but not always) valid input.

Reply:
I really like the brevity and it would be great to have some improvement to the above as I do think it is both too long and potentially confusing. But see https://github.com/astropy/astropy/pull/11118#issuecomment-813096479  for further discussion. I think this change, and more general docs refinement might be better saved for followup PRs. Here I've mostly been trying to just turn on x-ref and correct mistakes where I see them.

Reply:
@adrn I'm going to close these in favor of tackling them in a future PR. (https://community.openastronomy.org/t/abcd-always-be-code-documenting/56)

Reply:
Maybe put that list in a new issue?

Reply:
#11555